packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.controller.js

```diff
@@ -42,4 +58,79 @@ export default class TelecomPackMigrationCtrl {
         this.loading.init = false;
       });
   }
+
+  // Retrieve services from pack
+  getPackAccess() {
```

Comment:
Can't this be done in the `resolve` ?

Reply:
No because if there already is a task, the call to getPackAccess will not be done